lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -160,6 +170,26 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
     @Input()
     sidebarLeftTemplateContext: T = null;
 
+    /**
+     * Enable dialog box to allow user to download the previewed file, in case the preview is not responding for a set period of time.
+     * */
+    enableNonResponsiveDialog: boolean = false;
+
+    /**
+     * Enable reminder dialogs to prompt user to download the file, in case the preview is not responding for a set period of time.
+     * */
+    enableNonResponsiveDialogReminders: boolean = false;
+
+    /**
+     * Initial time in seconds to wait before giving the first prompt to user to download the file
+     * */
+    nonResponsivePreviewInitialTimerInSeconds: number = 50;
+
+    /**
+     * Time in seconds to wait before giving the second and consequent reminders to the user to download the file.
+     * */
+    nonResponsivePreviewReminderTimerInSeconds: number = 15;
```

Comment:
we already state in the documentation that it is in seconds, no need adding "Seconds" to the name

Reply:
Removed 'inSeconds'. Renamed property to `downloadPromptDelay` and `downloadPromptReminderDelay`